src/freexl.mk

```diff
@@ -0,0 +1,39 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := freexl
+$(PKG)_WEBSITE  := https://www.gaia-gis.it/fossil/freexl/index
+$(PKG)_DESCR    := FreeXL
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 1.0.3
+$(PKG)_CHECKSUM := f8ed29e03a6155454e538fce621e53991a270fcee31120ded339cff2523650a8
+$(PKG)_SUBDIR   := freexl-$($(PKG)_VERSION)
+$(PKG)_FILE     := freexl-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := http://www.gaia-gis.it/gaia-sins/freexl-sources/$($(PKG)_FILE)
+$(PKG)_DEPS     := gcc libiconv
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- 'http://www.gaia-gis.it/gaia-sins/freexl-sources/' | \
+    $(SED) -n 's,.*freexl-\([0-9][^>]*\)\.tar.*,\1,p' | \
+    tail -1
+endef
+
+define $(PKG)_BUILD
+    cd '$(1)' && ./configure \
```

Comment:
Try to use out-of-source builds (like qtkeychain below) using `$(BUILD_DIR)` and `$(SOURCE_DIR)` instead of `$(1)`